### PR TITLE
Cleanup scripts

### PR DESCRIPTION
**Pull request by piyush-J**

### CODE REVIEW

File: scripts/infer_v2.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```
Comment (piyush-J) [inline scripts/infer_v2.sh]: Delete this file

File: scripts/run-dljc.sh [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+WORKING_DIR=$(pwd)
+
+JSR308=$JSR308
+
+DLJC="$JSR308"/do-like-javac
+export AFU="$JSR308"/annotation-tools/annotation-file-utilities
+export PATH="$PATH":"$AFU"/scripts
+export CFI="$JSR308"/checker-framework-inference
+
+export CLASSPATH="$JSR308"/universe/bin/src:$CHECKERFRAMEWORK/dataflow/build:$CHECKERFRAMEWORK/javacutil/build:$CHECKERFRAMEWORK/stubparser/build:$CHECKERFRAMEWORK/framework/build:$CHECKERFRAMEWORK/checker/build:$SOLVER/bin:$CHECKERFRAMEWORK/framework/tests/junit-4.12.jar:$CHECKERFRAMEWORK/framework/tests/hamcrest-core-1.3.jar:$CFI/dist/checker-framework-inference.jar:$CFI/dist/org.ow2.sat4j.core-2.3.4.jar:$CFI/dist/commons-logging-1.2.jar:$CFI/dist/log4j-1.2.16.jar:$JSR308/jsr308-langtools/build/classes:$CLASSPATH
```
Comment (piyush-J) [inline scripts/run-dljc.sh]: Use setup.sh instead of this duplication

File: scripts/setup.sh [hunk 3]
```diff
@@ -1,14 +1,11 @@
 #!/bin/bash
 export MYDIR=`dirname $0`
 
-# Universe top level directory
-export UNIVERSE=$MYDIR/..
+# Playground top level directory
```
Comment (wmdietl) [inline scripts/setup.sh]:
```suggestion
# Universe top level directory
```

File: scripts/setup.sh [hunk 4]
```diff
@@ -1,14 +1,11 @@
...
 # CheckerFramework Inference
-export CFI="$JSR308"/checker-framework-inference
-
-# Generic Type Inference Solver
-export SOLVER="$JSR308"/generic-type-inference-solver
+export CFI="$MYDIR"/../../checker-framework-inference
 
 # Dependencies
-export CLASSPATH=$UNIVERSE/bin:$CHECKERFRAMEWORK/dataflow/build:$CHECKERFRAMEWORK/javacutil/build:$CHECKERFRAMEWORK/stubparser/build:$CHECKERFRAMEWORK/framework/build:$CHECKERFRAMEWORK/checker/build:$SOLVER/bin:$CHECKERFRAMEWORK/framework/tests/junit-4.12.jar:$CHECKERFRAMEWORK/framework/tests/hamcrest-core-1.3.jar:$CFI/bin:$CFI/dist/org.ow2.sat4j.core-2.3.4.jar:$JSR308/jsr308-langtools/build/classes:$CLASSPATH
+export CLASSPATH="$UNIVERSE"/bin/src:$CHECKERFRAMEWORK/dataflow/build:$CHECKERFRAMEWORK/javacutil/build:$CHECKERFRAMEWORK/stubparser/build:$CHECKERFRAMEWORK/framework/build:$CHECKERFRAMEWORK/checker/build:$CHECKERFRAMEWORK/framework/tests/junit-4.12.jar:$CHECKERFRAMEWORK/framework/tests/hamcrest-core-1.3.jar:$CFI/bin:$CFI/dist/org.ow2.sat4j.core-2.3.4.jar:$CFI/dist/commons-logging-1.2.jar:$CFI/dist/log4j-1.2.16.jar:./build/libs/universe.jar:$CFI/dist/checker-framework-inference.jar:$CLASSPATH
```
Comment (wmdietl) [inline scripts/setup.sh]: I think several of these entries can be removed.
- `"$UNIVERSE"/bin/src` is odd. What is in the `bin/src` directory that should be on the classpath? I think `bin` used to be the output directory of Eclipse. I don't think it needs to be added.
- The `build` directories are probably not correct: the class files are further down in the hierarchies. This needs to either be updated or we remove these and just require the .jar files be built.
- are the junit and hamcrest class files not contained in `checker-framework-inference.jar`? I assume the latter contains everything from `checker.jar`, but maybe not these test files.

**Review by wmdietl — CHANGES_REQUESTED** [review]

File: scripts/check.sh [hunk 5]
```diff
@@ -1,7 +1,41 @@
 #!/bin/bash
-
+set -e
```
Comment (wmdietl) [inline scripts/check.sh:2]: For consistency, should we add this to all scripts to quit on errors?
(Also, line 3 was removed or is absent in other files, so could be removed here, too)

File: scripts/debug.sh [hunk 6]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```
Comment (wmdietl) [inline scripts/debug.sh]: Maybe `debug-infer.sh` would be a better name for this script?

File: scripts/run-dljc.sh [hunk 7]
```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+WORKING_DIR=$(pwd)
+
+export MYDIR=`dirname $0`
+. ./$MYDIR/setup.sh
+
+JSR308=$JSR308
```
Comment (wmdietl) [inline scripts/run-dljc.sh]: This variable isn't used in other scripts and isn't set in setup.sh.
Can you please remove it and just do it relative to `MYDIR`, like it's done in other scripts?

File: scripts/setup.sh [hunk 8]
```diff
@@ -2,13 +2,10 @@
...
 # CheckerFramework Inference
-export CFI="$JSR308"/checker-framework-inference
-
-# Generic Type Inference Solver
-export SOLVER="$JSR308"/generic-type-inference-solver
+export CFI="$MYDIR"/../../checker-framework-inference
 
 # Dependencies
-export CLASSPATH=$UNIVERSE/bin:$CHECKERFRAMEWORK/dataflow/build:$CHECKERFRAMEWORK/javacutil/build:$CHECKERFRAMEWORK/stubparser/build:$CHECKERFRAMEWORK/framework/build:$CHECKERFRAMEWORK/checker/build:$SOLVER/bin:$CHECKERFRAMEWORK/framework/tests/junit-4.12.jar:$CHECKERFRAMEWORK/framework/tests/hamcrest-core-1.3.jar:$CFI/bin:$CFI/dist/org.ow2.sat4j.core-2.3.4.jar:$JSR308/jsr308-langtools/build/classes:$CLASSPATH
+export CLASSPATH=$CFI/bin:$CFI/dist/org.ow2.sat4j.core-2.3.4.jar:$CFI/dist/commons-logging-1.2.jar:$CFI/dist/log4j-1.2.16.jar:./build/libs/universe.jar:$CFI/dist/checker-framework-inference.jar:$CLASSPATH
```
Comment (wmdietl) [inline scripts/setup.sh]: Does `$CFI/bin` contain compiled `.class` files? I think we should remove this and, if at all, add the directories where gradle puts its output.

**Review by wmdietl — COMMENTED** [review]
Thanks! A few small comments left.

**Review by wmdietl — APPROVED** [review]

**Pull request merged**